calico_kubernetes/calico_kubernetes.py

```diff
@@ -347,7 +359,8 @@ def _get_container_info(self, container_name):
             info = self._docker_client.inspect_container(container_name)
         except APIError as e:
             if e.response.status_code == 404:
-                logger.error("Container %s was not found. Exiting.", container_name)
+                logger.error(
```

Comment:
There are a number of logs that are formatted like this in this PR.

Could you update the formatting to look a bit nicer, like this for example:

```
logg.error("Container %s was not found. Exiting.", 
                  container_name)
```

or 

```
logger.error("This is a really long error message that probably" \
                    "needs to go on two lines")
```